### DISCUSSION TITLE
does the dashboard support Localization?

### DISCUSSION THREAD

**Discussion by liuliang-wt** [elsa-workflows/elsa-core#4423] [Q&A]

**Comment by sfmskywalker** [comment]
Hi @liuliang-wt,

Thank you for bringing up the topic of localization for the dashboard. As of now, Elsa Studio does not have built-in support for localization. However, I'd like to assure you that it's on our roadmap and we recognize the importance of this feature for our global user base. We'll keep the community updated on any progress in this area.

Thanks for your patience and understanding.
